e2e/paths/enterInternationalElements_test.js

```diff
@@ -0,0 +1,29 @@
+const config = require('../config.js');
+
+Feature('Enter international elements');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	I.waitForElement('.tabs', 10);
+	caseViewPage.goToNewActions(config.applicationActions.enterInternationalElement);
+	I.waitForElement('ccd-case-edit-page', 10);
+});
+
+Scenario('test form save with no details entered', (I) => {
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterInternationalElement}`);
+});
+//
+// Scenario('test half form filled', (I, enterInternationalElementsPage) => {
```

Comment:
Please remove commented out code.

Reply:
Actually we need test that tests data can be entered and then is visible in one of the view tabs.